packages/instantsearch.js/src/middlewares/createInsightsMiddleware.ts

```diff
@@ -170,6 +189,8 @@ export function createInsightsMiddleware<
         }
       },
       started() {
+        if (!canInit) return;
```

Comment:
is it better to have a middleware that does nothing (like this), or to just make it work without canInit, continuing to assume you have to call `init` yourselves.

I'd just remove this early return, and likely everything works fine, right?